Simplenote/Classes/NoteOptionsViewController.swift

```diff
@@ -83,38 +115,46 @@ class NoteOptionsViewController: UITableViewController {
+                configuration: { [weak self, note] (cell: UITableViewCell, row: Row) in
                     let cell = cell as! SwitchTableViewCell
                     cell.textLabel?.text = NSLocalizedString("Pin to Top", comment: "Note Options: Pin to Top")
                     cell.cellSwitch.addTarget(self, action: #selector(self?.handlePinToTop(sender:)), for: .primaryActionTriggered)
-                    cell.cellSwitch.accessibilityHint = NSLocalizedString("Tap to toggle pin to top", comment: "Accessibility hint for toggling pin to top")
+                    cell.cellSwitch.accessibilityLabel = NSLocalizedString("Pin toggle", comment: "Switch which marks a note as pinned or unpinned")
+                    cell.cellSwitch.accessibilityHint = note.pinned ?
+                        NSLocalizedString("Unpin note", comment: "Action to mark a note as unpinned") :
+                        NSLocalizedString("Pin note", comment: "Action to mark a note as pinned")
+                    cell.cellSwitch.isOn = note.pinned
                 }
             ),
             Row(style: .Switch,
-                configuration: { [weak self] (cell: UITableViewCell, row: Row) in
+                configuration: { [weak self, note] (cell: UITableViewCell, row: Row) in
                     let cell = cell as! SwitchTableViewCell
                     cell.textLabel?.text = NSLocalizedString("Markdown", comment: "Note Options: Toggle Markdown")
                     cell.cellSwitch.addTarget(self, action: #selector(self?.handleMarkdown(sender:)), for: .primaryActionTriggered)
-                    cell.cellSwitch.accessibilityHint = NSLocalizedString("Tap to toggle markdown mode", comment: "Accessibility hint for toggling markdown mode")
+                    cell.cellSwitch.accessibilityLabel = NSLocalizedString("Markdown toggle", comment: "Switch which marks a note as using Markdown formatting or not")
+                    cell.cellSwitch.accessibilityHint = note.markdown ?
+                        NSLocalizedString("Disable Markdown formatting", comment: "Accessibility hint for disabling markdown mode") :
+                        NSLocalizedString("Enable Markdown formatting", comment: "Accessibility hint for enabling markdown mode")
+                    cell.cellSwitch.isOn = note.markdown
                 }
             ),
             Row(style: .Value1,
                 configuration: { (cell: UITableViewCell, row: Row) in
                     let cell = cell as! Value1TableViewCell
                     cell.textLabel?.text = NSLocalizedString("Share", comment: "Note Options: Show Share Options")
-                    cell.accessibilityHint = NSLocalizedString("Tap to open share options", comment: "Accessibility hint on cell that activates share sheet")
+                    cell.accessibilityHint = NSLocalizedString("share-accessibility-hint", comment: "Accessibility hint on share button")
```

Comment:
There are a few localizations like this, but the standard all across the apps is your former approach. Mind switching back? (thank you!)